.github/workflows/core.yml

```diff
@@ -0,0 +1,131 @@
+name: Core
+
+# This workflow is triggered on pushes or pull request to the repository.
+on:
+  push:
+    # file paths to consider in the event. Optional; defaults to all.
+    paths:
+      - 'ci/**'
+      - 'knowhere/**'
+      - 'docker/build_env/**'
+      - '.github/workflows/core.yml'
+      - docker-compose.yml
+      - '!**.md'
+      - '!ci/jenkins/**'
+  pull_request:
+    # file paths to consider in the event. Optional; defaults to all.
+    paths:
+      - 'ci/**'
+      - 'knowhere/**'
+      - '.github/workflows/core.yml'
+      - docker-compose.yml
+      - '!**.md'
+      - '!ci/jenkins/**'
+
+jobs:
+  ubuntu:
+    name: AMD64 Ubuntu ${{ matrix.ubuntu }}
+    runs-on: ubuntu-latest
```

Comment:
this should change to `runs-on: ubuntu-${{ matrix.ubuntu }}` if you want to run on 18.04. the latest is 20.*